cfgov/v1/atomic_elements/molecules.py

```diff
@@ -212,7 +212,6 @@ class Meta:
 class CallToAction(blocks.StructBlock):
     slug_text = blocks.CharBlock(required=False)
     paragraph_text = blocks.RichTextBlock(required=False)
-    button = atoms.Button()
```

Comment:
I don't think this is right; note how the existing Button atom inherits from Hyperlink. Currently when you add a CTA, you can provide the button text, aria label, and URL, along with size:

![image](https://github.com/cfpb/consumerfinance.gov/assets/654645/2c645a37-1895-4e64-ae17-b334e41b5eb2)

Note also how the call-to-action.html template still makes use of e.g. `value.button.url`.

With the changes as proposed, it's no longer possible to specify those, and I think those values will get lost if existing pages with CTAs get new revisions.

Instead of removing this button entirely, I think we want to replace it with an `atoms.Hyperlink`. I think if you change this line to `button = atoms.Hyperlink()` you won't need a data migration, although you'll need to recreate the schema migrations here.

Reply:
Thanks!! Updated in https://github.com/cfpb/consumerfinance.gov/pull/7966/commits/a61a242412908387a8c39d82b54c15e466557e4c